christfellowshipchurch/apollos-app
# Fix change address

## DESCRIPTION

Fix the change of address

### How do I test this PR?
Go to your profile in app and attempt to change your address. It should be reflected in app and in Rock.

## SCREENSHOTS/GIFS

| iOS |
| --- | 
|![image](https://user-images.githubusercontent.com/2528817/114943860-463b0000-9e0c-11eb-8ccb-e81b9b6c4eef.png)|

## TODO:

- [ ] No new warnings in tests, storybook, or apps
- [ ] Assign relevant reviewers
- [ ] Upload GIF(s) of iOS, Android
- [ ] Get one peer approval before merging

## REVIEW:

**Manual QA**

- [ ] QA branch on iOS and ensure it looks/behaves as expected
- [ ] QA branch on Android and ensure it looks/behaves as expected

**Code Review**


> The purpose of PR Review is to _improve the quality of the software._
